Makefile.common

```diff
@@ -22,10 +23,10 @@ clean:
 
 install:
 	mkdir -p $(DESTDIR)$(PLUGIN_DIR_PURPLE)
-	install -m 664 $(TARGET) $(DESTDIR)$(PLUGIN_DIR_PURPLE)
+	install -m 0755 $(TARGET) $(DESTDIR)$(PLUGIN_DIR_PURPLE)
```

Comment:
I know what -p does. Why do you want to do it?

shared libraries on Linux do not need +x.